src/server--async.php

```diff
@@ -34,7 +36,7 @@
 }
 
 try {
-  $config = json_decode($configString, true);
+  $config = json_decode($configString, true, 512, JSON_THROW_ON_ERROR);
```

Comment:
New `JSON_THROW_ON_ERROR` is a better error experience with `json_decode` and `json_encode`